docs/_data/australia-2022-schedule.yaml

```diff
@@ -0,0 +1,93 @@
+talks_day1:
```

Comment:
This is all provisional right, as we're not publishing it?

Reply:
Yep